expo/gbe/views/bid_class_view.py

```diff
@@ -74,23 +74,25 @@ def BidClassView(request):
             conference = Conference.objects.filter(accepting_bids=True).first()
             new_class = form.save(commit=False)
             new_class.duration = Duration(minutes=new_class.length_minutes)
+            new_class.b_conference = conference
+            new_class.e_conference = conference
```

Comment:
because there is no longer a default value - we have to explictly set the conference.